linera-base/src/crypto.rs

```diff
@@ -747,6 +754,18 @@ fn le_bytes_to_u64_array(bytes: &[u8]) -> [u64; 4] {
     integers
 }
 
+/// Reads the `bytes` as four big-endian unsigned 64-bit integers and returns them.
+fn be_bytes_to_u64_array(bytes: &[u8]) -> [u64; 4] {
+    let mut integers = [0u64; 4];
+
+    integers[0] = u64::from_be_bytes(bytes[0..8].try_into().expect("incorrect indices"));
+    integers[1] = u64::from_be_bytes(bytes[8..16].try_into().expect("incorrect indices"));
+    integers[2] = u64::from_be_bytes(bytes[16..24].try_into().expect("incorrect indices"));
+    integers[3] = u64::from_be_bytes(bytes[24..32].try_into().expect("incorrect indices"));
+
+    integers
+}
+
 /// Returns the bytes that represent the `integers` in little-endian.
 fn u64_array_to_le_bytes(integers: [u64; 4]) -> [u8; 32] {
```

Comment:
Are we still using this? (And _should_ we be using this at all?)

Reply:
Might be less confusing to use the same endianness there, too.

Reply:
Please create a separate issue for that.